core-web/libs/sdk/angular/src/lib/layout/row/row.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+import { DotPageAssetLayoutRow } from '../../models';
+import { ColumnComponent } from '../column/column.component';
+
+@Component({
```

Comment:
This component, has to have any docs explaining what and why exists?.

Reply:
This component no, only the LayoutComponent is exposed